### ISSUE TITLE
1.5.2: Cannot read property 'split' of undefined as a constant issue.

### ISSUE THREAD

**Issue by AvatarX001** [bitpay/copay#3632]
I now see this every single time when opening Copay 1.5.2 on Windows 10.  Which in turns means the transaction screen appears blank.  

<img width="314" alt="copay - multisignature wallet 12_5_2015 10_31_49 pm" src="https://cloud.githubusercontent.com/assets/16231309/11699575/99c29044-9e8a-11e5-8930-b87c744b7fb4.png">

**Comment by matiu** [comment]
Could you try importing the backup in other platform and check if the problem persist?

**Comment by AvatarX001** [comment]
I think I might know what caused it. I moved some funds between my own addresses as a way to verify something to someone. This caused the [Moved] entry to appear in the transaction screen. After closing the wallet app and later on launching it again, is when I started to have that error.

Uninstalled and reinstalled and this didn't fixed the issue. Neither did erasing the transaction cache. 

The solution for me was to move the funds out to a newly created wallet within the Copay app. And then just erasing the old wallet.
